### ISSUE TITLE
Interval == 0 causes Warning: Error in colors_animal: object 'animal_name' not found

### ISSUE THREAD

**Issue by basille** [mablab/rpostgisLT#1]
Using the latest in `dev`, I still have issues with the `animal_name` in some cases. Super simple reproducible example: Load the stork dataset, and only change the interval unit to "days". The interface is immediately grayed out, and R returns the following message:

```
Warning: Error in colors_animal: object 'animal_name' not found
Stack trace (innermost first):
    82: colors_animal
    81: eval
    80: eval
    79: resolveFormula
    78: FUN
    77: lapply
    76: FUN
    75: lapply
    74: evalAll
    73: evalFormula
    72: invokeMethod
    71: eval
    70: eval
    69: %>%
    68: addPolylines
    67: function_list[[k]]
    66: withVisible
    65: freduce
    64: _fseq
    63: eval
    62: eval
    61: withVisible
    60: %>%
    59: observerFunc [./shiny/pgtrajPlotter.R#558]
     4: <Anonymous>
     3: do.call
     2: print.shiny.appobj
     1: <Promise>
```

Note that the function does not crash: reloading the page brings the app in a responsive state.

**Comment by balazsdukai** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [color error](https://trello.com/c/l6bdUQsO/55-color-error)

**Comment by balazsdukai** [comment]
fixed in 15248636152618b4f31a7fca697e51fb653f96bb

**Comment by balazsdukai** [comment]
Previous solution only works when interval==0 and the interval units are adjusted. When interval==0 and Next is clicked the app crashes.
